executor/show.go

```diff
@@ -128,113 +129,121 @@ func (e *ShowExec) Next(ctx context.Context, req *chunk.Chunk) error {
 }
 
 func (e *ShowExec) fetchAll(ctx context.Context) error {
+	var err error = nil
+
+	// Temporary disable select limit to avoid miss the result.
+	oldSelectLimit := e.ctx.GetSessionVars().SelectLimit
+	e.ctx.GetSessionVars().SelectLimit = math.MaxUint64
```

Comment:
Maybe using defer to recover oldSelectLimit is better.

Another question is I'm not very understand why we should set `SelectLimit` to math.MaxUint64

Reply:
Yes, Have changed restore SelectLimit logic with defer.

The math.MaxUint64 is used as default of SelectLimit and it means no limit on return result set row numbers.